pkg/db-repo/repository.go

```diff
@@ -357,6 +371,120 @@ func ignoreCreatedAtIfNeeded(scope *gorm.Scope) {
 	}
 }
 
+// parses the []uint8 (from mysql) for field x, if there is a field xSet with type []string
+func GormParseMysqlSetType(scope *gorm.Scope) {
+	vo := reflect.ValueOf(scope.Value)
+	if vo.Kind() == reflect.Invalid {
+		return
+	}
+	for vo.Kind() == reflect.Ptr {
+		vo = vo.Elem()
+	}
+	if vo.Kind() == reflect.Slice {
+		for i := 0; i < vo.Len(); i++ {
+			parseMysqlSetType(vo.Index(i))
```

Comment:
Can we have the case of a `[][]Foo`? If so, this is not sufficient, but can't think of anything like this from my head...

Reply:
i think we only either have Foo or []Foo, but never [][]Foo. Why? i would expect that the recursion would do just fine?

Reply:
If this whole block would be in `parseMysqlSetType`, it would not matter if you have `[]Foo` or `[][][][][][][]*[][]Foo`